spec/index.html

```diff
@@ -1489,6 +1502,253 @@ <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
 </section>
 
 
+<section id="section-classic-full-interop" class="informative">
+  <h2>Interoperability between RDF [=Classic=] and RDF [=Full=]</h2>
+
+  <p class=issue>Should we make this section normative?</p>
```

Comment:
Yes, I think it should be.

It should be the way of doing the mapping, not to be taken as an illustration/suggestion.